CHANGES.md

```diff
@@ -1,5 +1,26 @@
 # Changes
 
+## edge-21.5.2
+
+This edge release updates the proxy-init container to check whether the iptables
+rules have already been added, which prevents errors if the proxy-init container
+is restarted. Also, the linkerd cli now has tab completion for kubernetes
+resources, saving you precious keystrokes! Finally, the proxy has been updated
+with several fixes and improvements.
+
+* Added instructions to `build.md` for using a locally built proxy
+  (thanks @jroper!)
+* Added support for kubernetes resource aware tab completion
+* Updated `proxy-init` to skip configuring firewall if rules exists
+* Fixed viz uninstall to delete all RBAC objects (thanks @aryan9600!)
```

Comment:
If you look at the rest of this file we go back and forth on how an extension is called out, but I would go with any of the following: "linkerd-viz uninstall", "Viz uninstall", "`viz uninstall`", etc.

Basically making it obvious this is an extension.

```suggestion
* Fixed Viz uninstall to delete all RBAC objects (thanks @aryan9600!)
```

Reply:
This is a really good callout because the inconsistency can lead to confusion.

What do you think of:
- using `linkerd-viz` when referring to the extension itself (code, charts, files, architecture, etc.)
- using `viz <command>` when referring to a subcommand that can be run from the CLI?

Reply:
Yea that seems good to me! I remember discussing this before; it may have been in the original extensions RFC but I'm not positive. Either way this seems like a good way to refer to things.